### ISSUE TITLE
Sprite Size based on `allSprites.tileSize` is reset on removing collider

### ISSUE THREAD

**Issue by wThomas84** [molleindustria/p5.play#230]
Hello,

thanks for sharing this awesome library. Had quite some fun so far.
However, I think I found a bug and want to report it to you.

The scaling with `allSprites.tileSize` does not work on Sprites, having no collider or collider being removed (collider= 'none')

Find below the code to reproduce the issue (it is based on one of your examples from the [documentation ](https://p5play.org/learn/tiles.html?page=1).

Best regards
Thomas

```
let player, blocks;
let gridSize = 32;

function setup() {
	new Canvas(480, 192);

	allSprites.tileSize = gridSize;
	allSprites.rotationLock = true;

	blocks = new Group();
	new blocks.Sprite(4, 5);
	new blocks.Sprite(3, 2);


	let b3 = new blocks.Sprite(6, 3);
	b3.text='b3'
// This line causes block b3 to appear smaller, as if the tileSize is not taken into account
	b3.collider = 'none';
}

function draw() {
	clear();
	for (let i = 0; i < width / gridSize; i++) {
		for (let j = 0; j < height / gridSize; j++) {
			rect(i * gridSize, j * gridSize, gridSize, gridSize);
		}
	}
}
```

**Comment by quinton-ashley** [comment]
Thanks for the report, you're right that's a bug! I will fix it soon.

**Comment by quinton-ashley** [comment]
fixed in v3.2.18
